manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: the-sz.PreyHelper
+PackageVersion: 1.13

```

```suggestion
PackageVersion: "1.13"
```
Floating point number needs to be quoted to be compliant with YAML semantics